packages/spotlight/src/spotlight.js

```diff
@@ -482,6 +488,7 @@ const Spotlight = (function () {
 				window.addEventListener('keyup', onKeyUp);
 				window.addEventListener('mouseover', onMouseOver);
 				window.addEventListener('mousemove', onMouseMove);
+				document.addEventListener('webOSMouse', webOSMouseHandler);
```

Comment:
Let's add a check for `webOS` so that we don't have to add the `webOSMouse` handler needlessly.

You can do this by importing `import platform from '@enact/core/platform'`, then check `platform.webos`, and if that's true, attach the `webOSMouse` handler.